lua/telescope/builtin/files.lua

```diff
@@ -229,8 +230,19 @@ files.file_browser = function(opts)
     }
   end
 
+  local function get_prefix(path)
+    path = Path:new(vim.fn.fnamemodify(path, ':p'))
```

Comment:
I just realize we no longer need the `vim.fn.fnamemodify` after https://github.com/nvim-telescope/telescope.nvim/pull/613 is merged. I forget to normalize the path before but #613 has the fix for it.
With normalize i mean:
/home/conni/repos/../ -> /home/conni and i think thats what you want/need